data/strings/strings.txt

```diff
@@ -2754,7 +2754,7 @@
 
```

Comment:
Thank you for contribution! This is great contribution with the cool new feature!

This "right swipe" UX is not obvious, but it follows Apple's best practices. I am OK with the proposed UX.

For the further work, please also consider adding group operations (Move, Delete) via "Edit" menu (a separate PR, this one is already good).